package.json

```diff
@@ -3,9 +3,13 @@
 	"version": "1.0.0",
 	"description": "",
 	"scripts": {
-		"build": "cross-env NODE_ENV=production webpack --display-modules",
+		"build": "cross-env NODE_ENV=production webpack",
+		"build:dev": "cross-env NODE_ENV=production webpack --config-name dev",
+		"build:web": "cross-env NODE_ENV=production webpack --config-name web",
+		"build:zotero": "cross-env NODE_ENV=production webpack --config-name zotero",
 		"build:ios": "cross-env NODE_ENV=production webpack --config-name ios",
-		"start": "webpack-dev-server",
+		"start": "webpack serve --mode development",
+		"start:dev": "webpack serve --mode development --config-name dev",
```

Comment:
I think we can just have a single `start` option here that would run the `dev` build. Other builds are intended to work only in specific environment therefore running a development server isn't useful with them.